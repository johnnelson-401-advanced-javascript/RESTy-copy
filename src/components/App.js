import React from 'react';
import Header from './Header';
import Footer from './Footer';
import Main from '../containers/Main';
import style from './App.css';

export default function App() {

  return (
    <div className={style.App}>
      <Header />
      <Main />
      <Footer />
    </div>
  );
}
