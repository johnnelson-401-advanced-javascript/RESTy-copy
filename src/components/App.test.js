import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

describe('App Test', () => {
  it('Renders an App', () => {
    const wrapper = shallow(<App />);
    expect(wrapper).toMatchSnapshot();
  });
});
