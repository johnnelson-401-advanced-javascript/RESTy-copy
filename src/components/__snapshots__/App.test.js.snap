// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test Renders an App 1`] = `
<div
  className="App"
>
  <Header />
  <Main />
  <Footer />
</div>
`;
